packages/react-components/testing/karma.conf.js

```diff
@@ -34,6 +34,9 @@ module.exports = function karmaConfig(config) {
           '@department-of-veterans-affairs/web-components/react-bindings':
             path.resolve(__dirname, '../../web-components/'),
         },
+        fallback: {
+          "util": require.resolve("util/")
```

Comment:
> [!note]
> This was added to address the [react-components node.js error](https://github.com/department-of-veterans-affairs/component-library/actions/runs/15645401046/job/44081850440?pr=1642): 
> > ERROR in ../../node_modules/sinon/lib/sinon/util/core/format.js 3:14-37
Module not found: Error: Can't resolve 'util' in '/home/runner/work/component-library/component-library/node_modules/sinon/lib/sinon/util/core'